SS_write_ssnew.tpl

```diff
@@ -329,8 +329,15 @@ FUNCTION void write_nudata()
     // info on dimensions of tagging data
     report1<<N_TG<<" # N tag groups"<<endl;
     report1<<N_TG_recap<<" # N recap events"<<endl;
-    report1<<TG_mixperiod<<" # mixing latency period: N periods to delay before comparing observed to expected recoveries (0 = release period)"<<endl;
-    report1<<TG_maxperiods<<" # max periods (seasons) to track recoveries, after which tags enter accumulator"<<endl;
+    report1<<TG_mixperiod_rd<<" # TG_mixperiod (latency) is the first period in which logL is calculated for this tag group (0 = release period)"<<endl;
+    report1<<"#  (use negative value for TG_mixperiod to invoke reading TG_min_recap after TG_maxperiods"<<endl;
+    report1<<TG_maxperiods<<" # max periods (seasons) to track recoveries, after which tags are still read into this accumulator period"<<endl;
+    if(TG_mixperiod_rd>=0){
+      report1<<"#  TG_min_recap would go here if TG_mixperiod < 0"<<endl;
+    }
+    else{
+      report1<<TG_min_recap<<" # TG_min_recap:  N recaps >= mixperiod must be >= min_recap to include tag group in logL"<<endl;
```

Comment:
Is there a way to reword this more concisely? I am having trouble following the statements. Maybe just simpl put 
`# COND: TG_min_recap if TG_mixperiod < 0` to mirror the statement produced when the mix period is greater than 0?